### PR TITLE
Autoscaler WG: add Marcin Kubica

### PR DESCRIPTION
**Pull request by marcinkubica**
Add Marcin Kubica (work for SAP)

### CODE REVIEW

File: org/cloudfoundry.yml
Comment (silvestre) [inline org/cloudfoundry.yml:5792]: Unfortunately you do not yet meet the requirements for a maintainer as defined in https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#requirements-2 - Please add yourself to the `members` section

```suggestion
        members:
        - marcinkubica
```

**Review by silvestre — CHANGES_REQUESTED** [review]

**Comment by marcinkubica** [comment]
Hi @Gerg are you ok to merge this one for me?
Cheers

**Comment by Gerg** [comment]
Per https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0008-role-change-process.md#promotion-to-approver, you need 2 existing approvers to review your application and attest that you meet the criteria for the approver role. I see @silvestre already approved the promotion, so you will need one more.

Once that criteria is met, I'll merge this PR.

**Review by asalan316 — APPROVED** [review]
LGTM

**Comment by marcinkubica** [comment]
@Gerg thanks, done.

**Review by Gerg — APPROVED** [review]

**Pull request merged**
